### PR TITLE
Evidencemodeler new recipe

### PR DESCRIPTION
**Pull request by nextgenusfs**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/evidencemodeler/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+readonly EVM_HOME=${PREFIX}/opt/${PKG_NAME}-${PKG_VERSION}
+
+mkdir -p ${EVM_HOME}
+
+cp -Rp simple_example PerlLib EvmUtils evidence_modeler.pl ${EVM_HOME}
```
Comment (bgruening) [inline recipes/evidencemodeler/build.sh]: Shouldn't evidence_modeler.pl not be in PATH after installation? You could symlink it into PREFIX/bin

File: recipes/evidencemodeler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  number: 0
+
+requirements:
+  run:
+    - perl
+    - perl-carp
+
+test:
+  commands:
+    - "${PREFIX}/opt/{{ name|lower }}-{{ version }}/evidence_modeler.pl -debug 2>&1 | grep \"################# Evidence Modeler\""
```
Comment (bgruening) [inline recipes/evidencemodeler/meta.yaml]: now the test can become easier I assume

**Comment by kastman** [comment]
@nextgenusfs This looks good to merge - any reason not to? After review it looks good to me if it looks good to you...

**Comment by nextgenusfs** [comment]
Yes I think it is fine. Had some issues getting the test to work because of how evidence modeler displays menu/help. But I think last commit is good to go.

**Pull request merged**